library/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellStepsDefinitionProvider.java

```diff
@@ -85,17 +89,31 @@ public List<Pair<Step, RetryRule>> get(
         Pair.of(
             new CreateLogAnalyticsWorkspaceStep(armManagers, resourceNameProvider),
             RetryRules.cloud()),
-        Pair.of(new CreatePostgresqlDNSStep(armManagers, resourceNameProvider), RetryRules.cloud()),
         Pair.of(
-            new CreateVirtualNetworkLinkStep(armManagers, resourceNameProvider),
+            new CreatePostgresqlDNSZoneStep(armManagers, resourceNameProvider), RetryRules.cloud()),
```

Comment:
Postgres does some DNS setup too -- renamed the classes to disambiguate a bit. I didn't change any postgres logic.

Reply:
I agree -- and batch/relay private endpoints will probably look very similar. I'll experiment with code organization/re-use.